From bf1c985746c136b39ed0a60817242731a09d2a19 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sun, 12 Jan 2014 17:06:02 +0000
Subject: [PATCH] builder: Document how to set Japanese language support in
 Fedora 20.

This updates/fixes commit 7e8ad13e863a5c87e3da47288bcecdb88ca0e4cb.

(cherry picked from commit fcdea83c6b9dc94a565a819a9f42e0c4721166e8)
---
 builder/virt-builder.pod | 18 +++++++-----------
 1 file changed, 7 insertions(+), 11 deletions(-)

diff --git a/builder/virt-builder.pod b/builder/virt-builder.pod
index c1adfa8..f7eb9e9 100644
--- a/builder/virt-builder.pod
+++ b/builder/virt-builder.pod
@@ -831,31 +831,27 @@ See L<https://wiki.debian.org/Keyboard>.
 
 Most Linux distributions support multiple locale settings so that you
 can have guest messages printed in another language such as Russian.
+
 However there is no single setting which controls this, since extra
-packages may need to be installed to support console and X fonts and
+packages may need to be installed to support console and X fonts, and
 keyboard input methods.  The packages required, and their
 configuration is highly distro-specific, and it is outside the scope
 of virt-builder to do this.
 
-=begin comment
-
 This section contains examples for some common Linux distributions.
 
 =head3 Setting Japanese in Fedora 20
 
-XXX This does not work.
-
  virt-builder fedora-20 \
    --size 20G \
    --update \
-   --install @gnome-desktop --install @japanese-support \
+   --install @japanese-support \
+   --install @xfce \
+   --install xorg-x11-server-Xorg,xorg-x11-drivers,rsyslog \
+   --link /usr/lib/systemd/system/graphical.target:/etc/systemd/system/default.target \
    --firstboot-command 'localectl set-locale LANG=ja_JP.utf8' \
    --firstboot-command 'localectl set-keymap jp' \
-   --firstboot-command 'systemctl enable gdm.service' \
-   --firstboot-command 'systemctl start gdm.service' \
-   --link /usr/lib/systemd/system/graphical.target:/etc/systemd/system/default.target
-
-=end comment
+   --firstboot-command 'systemctl isolate graphical.target'
 
 =head2 LOG FILE
 
-- 
1.8.4.2

